lib/util/print.js

```diff
@@ -101,7 +101,7 @@ function printTreeTraversal(node, parentNode) {
     }
   }
   else {
-    return node.toString();
+    return node.toString({notation: 'fixed'});
```

Comment:
ie 'fixed' and not 'sometimes scientific' or whatever the default logic is. I changed this after one of the specs was returning scientific notation for a neg number over 6 digits